code/mob/living/carbon/human.dm

```diff
@@ -3177,7 +3177,7 @@
 						if (priority > 0)
 							priority = "[NewLoc.step_material]"
 						else if (priority < 0)
-							priority = src.shoes ? src.shoes.step_sound : "step_barefoot"
+							priority = src.shoes ? src.shoes.step_sound : (iscow(src) ? "step_wood" : "step_barefoot")
```

Comment:
It would nice to see this as a var on the mutant race but this is fine for now

Reply:
Done!